### PR TITLE
Add ability to ignore specific paths

### PR DESCRIPTION
**Pull request by nowendwell**
This adds the ability to specify paths to skip request logging.

### CODE REVIEW

File: src/LarametricsRouteServiceProvider.php [hunk 1]
```diff
@@ -20,10 +21,10 @@ public function boot()
     {
         if(config('larametrics.requestsWatched')) {
             Route::matched(function($routeMatched) {
-                //dd($routeMatched);
+                //dd($routeMatched->route);
```
Comment (johannesschobel) [inline src/LarametricsRouteServiceProvider.php]: i think, all `dd()` and comments should be removed..

**Review by johannesschobel — CHANGES_REQUESTED** [review]

**Comment by josteph** [comment]
@nowendwell Wildcard support?

**Comment by nowendwell** [comment]
@josteph Yep. It uses the Request::is() which in turn uses the Str::is() function which does a preg_match. 

Tested using '\*r\*' and then tried 'password/reset' and 'register' and both were skipped.

File: src/config/larametrics.php [hunk 2]
```diff
@@ -52,6 +52,19 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | By default, when requestsWatched is enabled, all requests are  watched.
+    | If you would like Larametrics to skip a path you can add it below.
+    |
+    */
+    'requestsToSkip' => [
+        'login'
+    ],
+
```
Comment (tanmuhittin) [inline src/config/larametrics.php:67]: Should we have an option for method type of request? An array similar to below could be used.
`'requestsToSkip'=>[
  'GET'=>'*',
  'POST'=>['login','comment']
]`
Or maybe we can use route names and route group names for this action.

**Comment by aschmelyun** [comment]
Super handy feature, thanks for getting it out!

**Comment by johannesschobel** [comment]
I would go for the route-name approach, as I think this would be more fool-proof..

**Pull request merged**